### ISSUE TITLE
Attach folder failed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#358]
```
I installed Keepnote 0.6.5 in Ubuntu 10.04 LTS (AMD64) and found the "Attach 
folder" function failed, while this function was proper in Keepnote 0.6.3.

I installed Keepnote 0.6.5 by using python setup procedure "$ python setup.py 
install".





```

Original issue reported on code.google.com by `leon.yu...@gmail.com` on 6 Nov 2010 at 10:13

**Comment by GoogleCodeExporter** [comment]
```
What was the error that you saw?  Does the error log say anything?  "Help > 
Show Error Log".
```

Original comment by `matt.rasmus@gmail.com` on 7 Nov 2010 at 1:19
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Hi Matt,There was no error reported. The only result was that the folder can 
not be attached. 
```

Original comment by `leon.yu...@gmail.com` on 8 Nov 2010 at 4:49

**Comment by GoogleCodeExporter** [comment]
```
Does the error still occur in keepnote 0.6.6?
http://rasm.ods.org/keepnote/download-test/


```

Original comment by `matt.rasmus@gmail.com` on 8 Nov 2010 at 12:12

**Comment by GoogleCodeExporter** [comment]
```
Hi Matt,
I installed your newly released version 0.6.7. The "Attach Folder" function was 
proper, after I uninstalled the "Import Folder Tree" extension  and reinstalled 
it.  
```

Original comment by `leon.yu...@gmail.com` on 26 Nov 2010 at 9:06

**Comment by GoogleCodeExporter** [comment]
```
great!  Thanks for the follow up.
```

Original comment by `matt.rasmus@gmail.com` on 26 Nov 2010 at 3:14
- Changed state: **Done**
